src/common/commandExecutor.ts

```diff
@@ -79,19 +85,11 @@ export class CommandExecutor {
         });
 
         result.stderr.on("data", (data: Buffer) => {
-            if (outputChannel) {
-                outputChannel.append(data.toString());
-            } else {
-                process.stderr.write(data);
-            }
+            Log.logMessage(data.toString(), outputChannel || process.stderr, /*formatMessage*/false);
```

Comment:
I'd use a different method. Currently we use Log.logError for fatal errors. Maybe we should rename that method to Log.logFatalError and create a new Log.logError method for this kind of uses.